panel/src/components/Layout/Box.vue

```diff
@@ -56,7 +56,7 @@ export default {
   background: var(--color-white);
   border-radius: var(--rounded);
   line-height: 1.25rem;
-  padding: 0.75rem 1rem;
+  padding: 0.5rem 1rem;
```

Comment:
I think in relation the horizontal padding looks a bit off now. Maybe this:

```suggestion
  padding: 0.5rem 0.75rem;
```

Reply:
You're right. `0.75rem` more harmonious with other components 👍